README.md

```diff
@@ -77,6 +77,11 @@ $ git lfs track "*.psd"
 more about this pattern syntax
 [here](https://git-scm.com/docs/gitattributes)).
 
+**Note**: the quotations surrounding the pattern are important. Otherwise
+the glob pattern will be interpreted first by your shell and expanded into matching
+existing files. Multiple individual file entries will be created in 
+`.gitattributes` voiding the glob pattern in the future.
```

Comment:
I definitely think this is an important thing we should note. It's a tricky thing for users and I've seen other people get it wrong in the past with surprising results, as you mentioned.

Having said that, we'd like to keep the README file as short and concise as possible, since people are less likely to read longer documents. What would you think about condensing this into a single sentence, like, "Note that the quotation marks surrounding the pattern are important to prevent the glob pattern from being expanded by the shell"?

Reply:
Sounds good: updated.